doc/source/examples/compose.rst

```diff
@@ -133,12 +133,12 @@ Utilizing FDA and UCL
 If you want to try out the FDA and UCL you first have to do some preparations.
 
 
-0. Run the quickstart setup with the :code:`oauth2` profile:
+0. Run the example compose setup with the :code:`oauth2` profile:
    :code:`docker compose --profile oauth2 up -d`.
 1. Sign into the keycloak admin panel and create a logprep user in the :code:`logprep` realm.
    Make sure that the user is part of the :code:`logprep-admin` group and has a password. If you
    choose a password other than :code:`logprep` you have to update the credentials file
-   :code:`quickstart/exampledata/config/credentials.yml`, such that the password of
+   :code:`examples/exampledata/config/credentials.yml`, such that the password of
    :code:`http://localhost:3001` and :code:`http://localhost:3002` reflects your choice.
 2. You have to login to the FDA with the previously created user and create a release, as well
    as your first logclass. It is also necessary to add an example event to this logclass in order
```

Comment:
should we maybe change the layout of this example to be consistent with the other examples?